### DISCUSSION TITLE
Wrong balance after wallet migration

### DISCUSSION THREAD

**Discussion by junjie9021** [Chia-Network/chia-blockchain#10938] [Support]
I'm migrating my wallet data and putting the directory `~/chia ` back up to the new machine. The current version is 1.3.1. Then I import the wallet mnemonic file and find that the balance of the wallet is wrong. At first, I thought I had to wait for the wallet to synchronize. After waiting for two days, I still found that the balance is wrong. What should I pay attention to after migration?

**Comment by loppefaaret** [comment]
try and close down the chia client, and manually remove the current wallet database from `~\.chia\mainnet\wallet\db\`
then open up the chia client again, and let the wallet resync a new wallet database from 0
if you are still having issues, let me know
